products/lion/models/product/sedat.sql

```diff
@@ -0,0 +1,24 @@
+{{ config(
+    materialized = 'table'
+) }}
+
+WITH int_sedat AS (
+    SELECT * FROM {{ ref("int_sedat") }}
+)
+
+SELECT
+    -- Complete 87-character fixed-width record
+    CONCAT(
+        SUBSTR(LPAD(COALESCE(lionkey, ''), 10, ' '), 1, 10),
```

Comment:
Did you look into how Finn is formatting .dats in? e.g. [here](https://github.com/NYCPlanning/data-engineering/blob/0f9810e00dcce3efb945bbb05d74aafd277cc87b/products/lion/models/product/dat/lion_dat_fields.sql#L11C5-L11C21)